vaas/vaas/cluster/models.py

```diff
@@ -14,7 +15,25 @@
     vcl_template_comment_validator, name_validator, dc_symbol_validator
 
 
-class LogicalCluster(models.Model):
+class AbsModelWithJsonField:
+
+    def _get_parsed_field(self, field: Optional[set], values: str) -> Set[str]:
+        if field is None:
+            try:
+                field = set(json.loads(values))
+            except:  # noqa
+                field = set()
+        return field
+
+    def _prepare_set_and_json(self, field: Union[List, Set]) -> Tuple[Set, str]:
```

Comment:
We use python 3.9 and the Union have shorthand form. Just "|".
```python
Union[List, Set] = List | Set
```
Using that shorthand form is recommended by python.

Reply:
![Zrzut ekranu 2023-03-29 o 08 27 19](https://user-images.githubusercontent.com/4157902/228445176-4b59e9f3-9f4b-42de-85e3-19ca32138f19.png)